src/dist/replication/common/replication_common.cpp

```diff
@@ -518,6 +520,11 @@ void replication_options::initialize()
                                                           "bulk_load_provider_root",
                                                           "bulk_load_provider_root",
                                                           "bulk load root on remote file provider");
+    max_concurrent_bulk_load_downloading_count =
```

Comment:
For new added configs, preffer to use the util introduced by https://github.com/XiaoMi/rdsn/pull/275